packages/qa/tests/features/studyUpdateTests/studyUpdateWithCpmsTest.e2e.ts

```diff
@@ -39,7 +122,36 @@ test.describe('Update study and save changes locally in SE @se_184', () => {
       await expect(sePage.locator('.govuk-inset-text')).toBeVisible()
     })
 
-    await test.step(`And I update the study details with proposed changes`, async () => {})
+    await test.step(`And I update the study details with proposed changes`, async () => {
+      async function fillStudyDates(dateType: string, dd: string, mm: string, yyyy: string) {
+        await sePage.locator(`#${dateType}Date-day`).fill(dd)
+        await sePage.locator(`#${dateType}Date-month`).fill(mm)
+        await sePage.locator(`#${dateType}Date-year`).fill(yyyy)
+      }
+
+      // await sePage.locator('#status-3').click()
```

Comment:
Minor - are these comments needed?

Reply:
yeah these are work in progress will address in next PR